_i18n/en/resources/user-guides/securely_purchase.md

```diff
@@ -1,81 +1,43 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-This is a guide to purchase and securely store Monero as of June 2017.
+## Step 1: Acquire Monero
+There are multiple ways to acquire Monero: You can mine it, you can exchange services or goods for it or you can convert other cryptocurrencies and fiat money to XMR. If you choos the latter, the most convenient way to do it is to use an exchange.
```

Comment:
"choose"

Also Oxford comma after "for it".

Reply:
Fixed. Thanks.